applications/common/backend/src/test/java/pl/ds/kyanite/common/components/models/table/TableComponentTest.java

```diff
@@ -37,25 +44,79 @@ public class TableComponentTest {
   @BeforeEach
   public void init() {
     context.addModelsForClasses(TableComponent.class);
-    context.load().json(requireNonNull(
-        Thread.currentThread().getContextClassLoader().getResourceAsStream("table.json")), PATH);
   }
 
   @Test
   void defaultTableComponentModelTest() {
-    TableComponent model = context.resourceResolver().getResource(PATH + "/default")
-        .adaptTo(TableComponent.class);
+    // given
+    importTestTableResource();
+
+    // when
+    TableComponent model = toTableModel(PATH + "/default");
+
+    // then
```

Comment:
Just a small refactor here because of adding a new test, but no changes besides this